src/Braille/BraillePicture.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { BrailleCharacter as Character, BrailleDot as Dot } from 'puzzle-lib';
+
+type Props = {
+  width: number,
+  character: Character
+};
+
+class BraillePicture extends React.Component<Props> {
+  constructor(props: Props) {

```

Comment:
I forget if this is actually required. Does it still work if you don't declare the constructor function?